package com.day4;

import java.io.IOException;
import java.util.Scanner;

public class Test1 {

	public static void main(String[] args) throws IOException {
		// TODO Auto-generated method stub
		
		Scanner sc = new Scanner(System.in);
		
		int num1, num2, result=0;
		
		char oper;
		
		System.out.print("첫번째 수?");
		num1 = sc.nextInt();
		
		System.out.print("두번째 수?");
		num2 = sc.nextInt();
		
		System.out.print("연산자?[+,-,*,/]");
		oper = (char) System.in.read();
		
		switch(oper){
		
		case '+':
			result = num1+num2;
			break;
		case '-':
			result = num1 - num2;
			break;
		case '*':
			result = num1 * num2;
			break;
		case '/':
			result = num1 /num2;
			break;
		}
		System.out.println("결과 : " + result);
		

	}

}
